MusicCloud/MusicCloud/Scenes/MainTabbarController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  MainTabbarController.swift
+//  MusicCloud
+//
+//  Created by thanh on 2/22/19.
+//  Copyright © 2019 thanh. All rights reserved.
+//
+
+import UIKit
+import Reusable
+import ESTabBarController_swift
+
+class MainTabbarController: ESTabBarController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configTabbar()
+    }
+    
+    private func configTabbar() {
```

Comment:
toàn bộ code trong đoạn này có thể làm gọn hơn hẳn nhé, cách đang làm hiện giờ toàn là copy and paste nên cần phải xem lại